src/components/AdminDashboard/Incidents.js

```diff
@@ -1,123 +1,23 @@
-    confirmRejectHandler,
-    approveAndRejectHandler,
-    confirmCancel,
     selected,
-    selectAll,
-    allSelected,
-    handlePerPageChange,
-    currentSet,
-    setUnapprovedIncidents,
-    setPageNumber,
-    unapprovedIncidents,
     setSelected,
-    formResponses,
-    setCurrList,
+    setUnapprovedIncidents,
+    incidents,
   } = props;
 
   return (
     <>
-      <div className="dashboard-top-flex">
-        {/* <div className="dashboard-top-input">
-          <input
-            type="checkbox"
-            name="select-all"
-            onChange={confirmApprove ? () => {} : selectAll}
-            checked={allSelected}
-          />
-          <label id="select-all-label" htmlFor="select-all">
-            Select All{' '}
-          </label>
-        </div> */}
-
-        <div className="dashboard-top-approve-reject">
-          {!confirmApprove && !confirmReject ? (
-            <button
-              disabled={selected.length < 1}
-              onClick={confirmApproveHandler}
-              className={
-                selected.length > 0 ? 'approve-reject-select' : 'hidden-button'
-              }
-            >
-              Approve
-            </button>
-          ) : (
-            <button
-              onClick={approveAndRejectHandler}
-              className={
-                selected.length > 0 ? 'approve-reject-select' : 'hidden-button'
-              }
-            >
-              Yes
-            </button>
-          )}
-          {!confirmApprove && !confirmReject ? (
-            <button
-              disabled={selected.length < 1}
-              onClick={confirmRejectHandler}
-              className={
-                selected.length > 0 ? 'approve-reject-select' : 'hidden-button'
-              }
-            >
-              Reject
-            </button>
-          ) : (
-            <button
-              onClick={confirmCancel}
-              className={
-                selected.length > 0 ? 'approve-reject-select' : 'hidden-button'
-              }
-            >
-              No
-            </button>
-          )}
-          <p
-            className={
-              !confirmApprove && !confirmReject
-                ? 'confirmation-message transparent'
-                : 'confirmation-message'
-            }
-          >
-            Are you sure?
-          </p>
-        </div>
-        {/* <div className="dashboard-top-page-number">
-          <label id="items-per-page" htmlFor="per-page-selector">
-            Items Per Page
-          </label>
-          <select
-            className="items-pp-select"
-            name="per-page-selector"
-            onChange={handlePerPageChange}
-          >
-            <option value="5">5</option>
-            <option value="10">10</option>
-            <option value="25">25</option>
-            <option value="50">50</option>
-            <option value="100">100</option>
-          </select>
-        </div> */}
-      </div>
-
```

Comment:
Just want to make sure you tested out creating an incident / approving an incident on this branch following these changes, I don't remember seeing that in the loom walkthroughs. Think it would be good to make sure that type of testing is done on this PR just in-case.

Reply:
Create New Incident doesn't work currently as is (on the production branch, as well as this one.) It gives a CORS error for mapquest... I put a try/catch around the mapquest stuff and it gives an error from the backend. I think getting that to work is beyond the scope of this PR. But I can add it if you want.